src/Data/Json/JTable/Internal.purs

```diff
@@ -0,0 +1,218 @@
+module Data.Json.JTable.Internal
+  ( renderJTableRaw, renderRows, renderThead, renderTbody, tsToRows, sortTree
+  , Tree(..), tPath, tWidth, tHeight, tKids
+  , Cell(..), cCursor, cWidth, cHeight, cJsonPrim
+  , JPath(..), Table(..)
+  , tFromJson, tMergeArray, widthOfPrimTuple, padTree
+  , cFromJson, cMergeObj, mergeObjTuple
+  , foldJsonP, toPrim, zipWithIndex, orelse, strcmp, _nattr, _cspan, _rspan
```

Comment:
I'd prefer if we only exported the ones which are *actually called* from outside the module, because a lot of these are helper functions and could just as easily be defined in a `where` clause (well, except for repetition).